internal/distro/rhel9/azure.go

```diff
@@ -75,7 +28,6 @@ func azureRhuiCommonPackageSet(t *imageType) rpmmd.PackageSet {
 			"grub2-efi-x64",
 			"grub2-pc",
 			"hyperv-daemons",
-			"insights-client",
```

Comment:
Did `insights-client` just disappear, or am I missing something?

Reply:
nope, it is added as part of `distroSpecificPackageSet()` on RHEL-9. This is different compared to RHEL-8.

Reply:
Ah, okay. :)